src/SDKs/Search/DataPlane/Microsoft.Azure.Search/Properties/AssemblyInfo.cs

```diff
@@ -1,11 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-
-using System.Reflection;
-
-[assembly: AssemblyVersion("9.0.0.0")]
-[assembly: AssemblyFileVersion("9.0.1.0")]
-
-[assembly: AssemblyTitle("Microsoft Azure Search Library")]
-[assembly: AssemblyDescription("Makes it easy to develop a .NET application that uses Azure Search.")]
```

Comment:
Where does `AssemblyDescription` come from now that these files are gone? That was useful when browsing NuGet packages IMO

Reply:
It comes from the Description property in the project.